Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1075,12 +1075,13 @@ final class StatusTableViewController: ChartsTableViewController {
                 vc.restoreUserActivityState(activity)
             }
         case let vc as InsulinDeliveryTableViewController:
-            vc.doseStore = deviceManager.loopManager.doseStore
+            vc.deviceManager = deviceManager
```

Comment:
This change was made so InsulinDeliveryTableViewController can create an instance of BolusViewController